### PR TITLE
Fix virtualized lists should never be nested

### PR DESCRIPTION
**Pull request by KedziaPawel**

### CODE REVIEW

File: src/screens/Dashboard/TransactionList.tsx [hunk 1]
```diff
@@ -54,20 +54,35 @@ export class TransactionList extends PureComponent<Props> {
...
-  render() {
+  renderTransactions = () => {
+    const sections = this.getSectionData();
+    return sections.map(section => {
+      const { data: transactions, title } = section;
+      return (
+        <View key={title}>
+          {this.renderSectionTitle(title)}
+          {transactions.map(t => (
+            <TransactionItem key={t.txid} item={t} onPress={this.onTransactionItemPress} />
```
Comment (Kamill90) [inline src/screens/Dashboard/TransactionList.tsx:65]: I don't get this change. A Virtualized list (SectionList / FlatList ) is necessary for lists with more items like TransactionList. The same mechanism is in ContactList or RecoveryTransactionListScreen. Rendering transactions in simple view is an antipattern.

**Review by Kamill90 — CHANGES_REQUESTED** [review]

File: src/screens/Dashboard/TransactionList.tsx (hunk 1 above)
Comment (KedziaPawel) [reply src/screens/Dashboard/TransactionList.tsx:65]: SectionList is nested here inside scrollView, so there is no performance gain here. https://nyxo.app/fixing-virtualizedlists-should-never-be-nested-inside-plain-scrollviews 
Zrzut ekranu 2020-08-19 o 09 31 47

Comment (Kamill90) [reply src/screens/Dashboard/TransactionList.tsx:65]: Great that you have found it. Please set ScreenTemplate noScroll prop to true here so SectionList is wrapped with View instead ScrollView.

Comment (KedziaPawel) [reply src/screens/Dashboard/TransactionList.tsx:65]: I tried this but unfortunately, it won't work as we weed scrollView to scroll down as the whole view can't fit in the size of the phone screen

Comment (Kamill90) [reply src/screens/Dashboard/TransactionList.tsx:65]: Have you tried to move everything above the TransactionList to its ListHeaderComponent prop?

Comment (KedziaPawel) [reply src/screens/Dashboard/TransactionList.tsx:65]: I didn't, seemed to be complicated to move the whole logic inside the header but I'll try to do it

**Comment by KedziaPawel** [comment]
Done in another PR

**Pull request closed without merging**